applications/crossbar/src/modules/cb_storage.erl

```diff
@@ -596,3 +628,13 @@ decode_json(RespBody) ->
             lager:debug("error body not JSON"),
             RespBody
     end.
+
+maybe_update_plans(Context) ->
+    maybe_update_plans(Context, cb_context:resp_status(Context)).
+
+maybe_update_plans(Context, 'success') ->
+    SavedDoc = cb_context:doc(Context),
+    _ = kzs_plan:handle_new(SavedDoc),
```

Comment:
@jamesaimonetti can we find an alternative for this ? maybe a kz_caches model for pqc ?